src/dtlssrtptransport.cpp

```diff
@@ -73,13 +84,14 @@ DtlsSrtpTransport::~DtlsSrtpTransport() {
 	srtp_dealloc(mSrtpOut);
 }
 
+
 bool DtlsSrtpTransport::sendMedia(message_ptr message) {
 	std::lock_guard lock(sendMutex);
 	if (!message)
 		return false;
 
 	if (!mInitDone) {
-		PLOG_WARNING << "SRTP media sent before keys are derived";
```

Comment:
This warning doesn't need a counter, and could actually be promoted to an error.

It shouldn't happen as tracks should not have a transport before they are open by `PeerConnection`, which happens after the init here.